core/domain/image_services.py

```diff
@@ -22,17 +22,22 @@
 import io
```

Comment:
Should we move this to the image_validation_services.py file and rename it to image_services (or vice versa)?

Reply:
Per discussion with other contributors, we thought of having two separate files (image_validation_service more around Appia's requirement and image_services more around general methods related to images. We have a similar pattern for HTML services (html_validation_services & html_cleaner). If required in future we we will merge these kinda file into one!) 

No changes required for this comment, I'll mark this comment "Resolved".